package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "o-grid",
+  "version": "1.0.0",
+  "description": "A 12 column responsive, flexbox-based grid system for laying out documents, templates and modules.",
+  "main": "main.js",
+  "devDependencies": {
+    "mocha": "^5.0.0",
```

Comment:
Mocha 5 dropped support for IE9 and IE10 --> https://github.com/mochajs/mocha/releases/tag/v5.0.0